docs/contracts/inheritance.rst

```diff
@@ -181,8 +181,9 @@ Function Overriding
 ===================
 
 Base functions can be overridden by inheriting contracts to change their
-behavior. The overriding function must then use the ``override`` keyword in the
-function header as shown in this example:
+behavior if they are marked as ``virtual`` and have a non-``private``
```

Comment:
Maybe just add the private special case at the end in a note? I mean this information is not really important because everyone would use it like that anyway, and if it is part of the initial sentence, it might be too distracting.